apier/v1/apier_it_test.go

```diff
@@ -833,7 +833,7 @@ func testApierLoadAccountActions(t *testing.T) {
 func testApierReloadScheduler(t *testing.T) {
 	var reply string
 	// Simple test that command is executed without errors
-	if err := rater.Call(utils.SchedulerSv1Reload, dispatchers.StringWithApiKey{}, &reply); err != nil {
+	if err := rater.Call(utils.SchedulerSv1Reload, utils.StringWithApiKey{}, &reply); err != nil {
```

Comment:
the utils.StringWithApiKey should be a pointer, best to be done as: new(utils.StringWithApiKey). Please review everywhere in APIs.